api/src/test/java/org/openmrs/AllergenTest.java

```diff
@@ -35,7 +53,7 @@ public void shouldEitherBeCodedOrFreeText() {
 	}
 	
 	private void assertCoded() {
-		Assert.assertNotEquals(allergen.getCodedAllergen().getUuid(), Allergen.OTHER_NON_CODED_UUID);
+		Assert.assertNotEquals(allergen.getCodedAllergen().getUuid(), otherNonCodedConceptUuid);
```

Comment:
Shouldn't you be using Allergen.getOtherNonCodedConceptUuid() instead of the local variable otherNonCodedConceptUuid?